README.md

```diff
@@ -1,48 +1,117 @@
+the web fonts implementation that was used in version 4 and older of Font Awesome. You might head over there to learn about how it works.
+
+> https://fontawesome.com/how-to-use/svg-with-js
+
+#### Going from 0.0.x to 0.1.0
+
+See [UPGRADING.md](./UPGRADING.md).
+
+You might also be interested in the larger umbrella project [UPGRADING.md](https://github.com/FortAwesome/Font-Awesome/blob/master/UPGRADING.md)
+
+## Installation
+
+This project is an Ember addon. So we'll add that first:
+
+```
+$ ember install @fortawesome/ember-fontawesome
 ```
 
-To support that use, you'd need to:
+We need at least one style. Let's start with the free version of Solid.
 
-1. `ember install @fortawesome/ember-fontawesome`
+```
+$ npm i --save @fortawesome/free-solid-svg-icons
+```
 
-1. `yarn add @fortawesome/fontawesome-free-solid` (or `npm install ...`). 
-Do this for each icon pack you'll use in your app. By default, all installed icon packs will be bundled into
-`vendor.js` and also added to the Font Awesome library (i.e. `library.add()`)
+## Add more styles or Pro icons
 
-(NOTE: As of Feb 14, 2018 it's best to use the 5.1 prereleases of the icon packs, since you're already living
-on the edge. So, `yarn add @fortawesome/fontawesome-free-solid@5.1.0-3`)
+Brands are separated into their own style and for customers upgrading from
+version 4 to 5 we have a limited number of Regular icons available.
 
-1. If you want to include only a subset of icons from an icon pack, add a `fontawesome` configuration 
-object to your applications options in `ember-cli-build.js`. The following example declares that all 
-icons in fontawesome-free-solid should be included in the vendor.js bundle add added to the library,
-and for fontawesome-pro-light, only faAdjust and faAmbulance are to be included in the bundle and added to the library.
+**Visit [fontawesome.com/icons](https://fontawesome.com/icons) to search for free and Pro icons**
 
 ```
-  // ...
-  let app = new EmberApp(defaults, {
-    // Add options here
-    fontawesome: {
-      icons: {
-        'fontawesome-free-solid': 'all'
-        'fontawesome-pro-light': [
-          'faAdjust',
-          'faAmbulance'
-         ]
-      }
-  })
+$ npm i --save-dev @fortawesome/free-brands-svg-icons
+$ npm i --save-dev @fortawesome/free-regular-svg-icons
+```
+
+Do this for each icon pack you'll use in your app. By default, all installed
+icon packs will be bundled into `vendor.js` and also added to the Font Awesome
+library (i.e. `library.add()`)
+
+If you are a [Font Awesome Pro](https://fontawesome.com/pro) subscriber you can install Pro packages.
+
+```
+$ npm i --save-dev @fortawesome/pro-solid-svg-icons
+$ npm i --save-dev @fortawesome/pro-regular-svg-icons
+$ npm i --save-dev @fortawesome/pro-light-svg-icons
+```
+
+Using the Pro packages requires [additional configuration](https://fontawesome.com/how-to-use/js-component-packages).
+
+## or with Yarn
+
+```
+$ yarn add --dev @fortawesome/fontawesome-svg-core
```

Comment:
is there a reason to install the `core` lib manually, or should this match the other examples and just contain an icon pack?

Reply:
There is not. Will fix.